src/ucp/api/ucp.h

```diff
@@ -2774,10 +2774,6 @@ ucs_status_ptr_t ucp_stream_recv_nb(ucp_ep_h ep, void *buffer, size_t count,
  * complete when the message is delivered to the buffer. If the receive
  * operation cannot be started, the routine returns an error.
  *
- * @note This operation supports specific flags, which can be passed
```

Comment:
need to move this to `@param` doc

Reply:
will not work - flags are added in separate PR
will update wording after https://github.com/openucx/ucx/pull/4984 is merged

Reply:
updated wording